projects/imagemagick/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER paul.l.kehrer@gmail.com
+RUN apt-get update && apt-get install -y make autoconf automake libtool curl
+RUN git clone --depth 1 https://github.com/imagemagick/imagemagick
+RUN curl -O http://lcamtuf.coredump.cx/afl/demo/afl_testcases.tgz
```

Comment:
I think you don't need `curl`, see `ADD` command usage example: https://github.com/google/oss-fuzz/blob/master/projects/libreoffice/Dockerfile#L25